### PR TITLE
Uncomment more test_model pytests

### PR DESCRIPTION
**Pull request by ChrisJar**
Note: can't uncomment `test_show_models` because of nested keyword arg parsing issue

### CODE REVIEW

**Comment by jdye64** [comment]
@ChrisJar this PR is marked as draft but seems to have working tests? Is it ready for review?

**Comment by ChrisJar** [comment]
There's a few more things on the way

File: dask_planner/src/sql/logical/export_model.rs [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                }
+            }
+        }
+        Ok(options)
+    }
+}
+
+impl PyExportModel {
+    /// Given a SqlParserExpr instance retrieve the String value from it
+    fn _str_from_expr(expression: SqlParserExpr) -> String {
```
Comment (jdye64) [inline dask_planner/src/sql/logical/export_model.rs:101]: Could you move this whole function to `dask_planner/src/sql/parser_utils.rs`? Seems handy for others to use as well.

**Review by jdye64 — CHANGES_REQUESTED** [review]
Lots of good stuff here @ChrisJar couple of small requests but looks good!

**Review by jdye64 — APPROVED** [review]
LGTM

**Pull request merged**